gulpfile.js

```diff
@@ -40,6 +41,18 @@ gulp.task('serve:dist', ['build:dist'], require('./tasks/server')(gulp, config.d
 gulp.task('serve', ['compile:src', 'less', 'fonts', 'html', 'json', 'assets'],
     require('./tasks/server')(gulp, config.srcDir, config.watchDir, true, config.projectDir, config));
 
+gulp.task('typedoc', function() {
+  return gulp
+    .src(['src/services/**/*.ts'])
```

Comment:
Why don't we just create the docs for all of our src files? That way we have everything